### ISSUE TITLE
Multiple Trackers?

### ISSUE THREAD

**Issue by oivio** [TrueOpenVR/TrueOpenVR-Core#5]
Hey, once again here with a two small question. Is it possible to use multiple trackers with TOVR for Full Body Tracking?
I mean, currently, you can use expensive Valve Trackers also I guess Drive4vr have the ability to track light balls for full body tracking or it emulates Vive Trackers for Kinect. I was wondering if there is the ability to use for example does light balls or maybe even some addition to controlling joints movement etc? 
The last question, is it possible to send tracking data like position to JSON file or txt file and send that data from there to SteamVR? 

Since I am just a web developer the last question might help me to improve your amazing work.

**Comment by r57zone** [comment]
At this point in TrueOpenVR can track only the head and hands. There is no support for tracking the body at the API level. The body is tracked probably only in a pair of games, such as VR chat.

This can be done in a separate SteamVR driver, where it will work. Thus can replace the Vive trackers with ping-pong balls with led and ps3 cams.

At the moment, there is no driver with the TCP/UDP port to receive json packets. There is a Keyboard driver that allows to change the position with the buttons. There is also a FreeTrack driver, which can be controlled from OpenTrack (UDP input support) or [FreePie](https://github.com/r57zone/VR-tracking-apps/blob/master/FreePIE%20scripts/FreePIE_RazorIMU_PSMoveService.py#L10-#L17). 
This way you can change the position of the HMD, but it's not really necessary in games. In games, only the position of the controllers is important. Emulation the position of the controllers for the game with the buttons, but it's impossible to play in this way, so probably this does not make much sense. There is an [AndroidControllers driver](https://github.com/TrueOpenVR/TrueOpenVR-Drivers/blob/master/C%2B%2B/AndroidControllers/), it allows you to change the rotation and click on the buttons of the controllers, but does not change the position. The driver opens UDP ports 5555 and 5556.

By the way, there is simply a [SteamVR driver with UDP input](https://github.com/r57zone/OpenVR-OpenTrack), which allows you to change the position of the HMD.

**Comment by oivio** [comment]
Wow, thank you for really quick replay. To be honest I was actually aiming for VrChat. 
I did also ask since I am working on Kinect v1 and v2  integration and currently, I did manage to display each joint position with rotation via browser by using WebSockets.
I did read somewhere that LeapMotion driver works on the SteamVR backend by using some javascript. So I might need to have looked right there. 
My end goal is to make it work without needing extra software to work in the background. Make it simple as you did with TrueOpenVr.

Anyways, once again! Thank you so much!
